### PR TITLE
TINKERPOP-2999 3.7.0 Remote Console Sends Incomplete Queries

### PR DESCRIPTION
**Pull request by ryn5**
https://issues.apache.org/jira/browse/TINKERPOP-2999
Fixes issue where multiline commands would throw error on the first line, caused by upgrade to Groovy 4.  Handling pulled from Groovysh.groovy

### CODE REVIEW

File: gremlin-console/src/main/groovy/org/apache/tinkerpop/gremlin/console/GremlinGroovysh.groovy [hunk 1]
```diff
@@ -140,4 +171,48 @@ class GremlinGroovysh extends Groovysh {
...
+            })
+        }
+        // Evaluate the current buffer w/imports and dummy statement
+        List<String> buff
+        if (variableBlocks) {
+            buff = [importsSpec] + ['try {', 'true'] + current + ['} finally {' + variableBlocks + '}']
+        } else {
+            buff = [importsSpec] + ['true'] + current
+        }
+        setLastResult(result = interp.evaluate(buff))
```
Comment (kenhuuu) [inline gremlin-console/src/main/groovy/org/apache/tinkerpop/gremlin/console/GremlinGroovysh.groovy:198]: I'm a bit concerned here that we just do setLastResult() to whatever is in the buffer here. Notice how for remote console we normally do this with something like `mediator.currentRemote().submit([script])`. Have you checked to see if this is a problem?

Comment (ryn5) [reply gremlin-console/src/main/groovy/org/apache/tinkerpop/gremlin/console/GremlinGroovysh.groovy:198]: Confirmed that multiline queries are fixed for both interpreter modes

**Comment by Cole-Greer** [comment]
Thanks Ryan, LGTM.

VOTE +1

**Comment by kenhuuu** [comment]
VOTE +1

**Comment by xiazcy** [comment]
VOTE +1

**Pull request merged**